Update CircuitPython Essentials Pin Map Script for Pico W

anecdata opened the pull request:
Also add a commented line to show `microcontroller` pins.

Requested by a user on Discord: https://discord.com/channels/327254708534116352/537365702651150357/1074369255941877921

dhalbert commented on CircuitPython_Essentials/Pin_Map_Script/code.py:
```diff
@@ -1,15 +1,22 @@
 board_pins = []
 for pin in dir(microcontroller.pin):
-    if isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin):
+    if isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or \
+       (cyw43 and isinstance(getattr(microcontroller.pin, pin), cyw43.CywPin)):
```
```suggestion
    if (isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or 
        (cyw43 and isinstance(getattr(microcontroller.pin, pin), cyw43.CywPin))):
```

dhalbert reviewed (changes requested):
Just a formatting change.

dhalbert commented on CircuitPython_Essentials/Pin_Map_Script/code.py:
```diff
@@ -1,15 +1,22 @@
 board_pins = []
 for pin in dir(microcontroller.pin):
-    if isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin):
+    if (isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or 
```
```suggestion
    if (isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or
```

anecdata commented:
Is there more to do here? Dan made the changes and it's better now. Change requests appear to be resolved, but Github still doesn't look happy.

anecdata commented:
fixed conflict, and removed commented-out lines to show microcontroller pins since that was redundant with the added lies to show original GPIO parenthetically

(history of this little code snippet, as close as I could reconstruct it, is [here](https://gist.github.com/anecdata/1c345cb2d137776d76b97a5d5678dc97))

dhalbert reviewed (approved):
Looks good - thanks!
